File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if nums == []: return 0
        nums = list(set(nums))
        nums.sort()
        curr = 1
        m = 1
        for i in range(1, len(nums)):
            if nums[i] == nums[i-1] + 1:
                curr+=1
                m = max(curr, m)
            else:
                curr = 1
        return m

File: Data Structures & Algorithms/max-consecutive-ones-iii/submission-3.go
func longestOnes(nums []int, k int) int {
   var maxcount, l, replace int
   for r, rval := range(nums){
	   if rval == 0 {
		   replace++
	   }
	   for replace > k{
		   if nums[l] == 0 {
			   replace--
		   }
           l++
	   }
	   maxcount = max(maxcount, r-l+1)
   }
   return maxcount
}
